astropy/units/format/generic_parsetab.py

```diff
@@ -1,77 +1,76 @@
-  ('product_of_units -> unit_expression product_of_units','product_of_units',2,'p_product_of_units','astropy/units/format/generic.py',204),
-  ('product_of_units -> unit_expression','product_of_units',1,'p_product_of_units','astropy/units/format/generic.py',205),
-  ('unit_expression -> function','unit_expression',1,'p_unit_expression','astropy/units/format/generic.py',216),
-  ('unit_expression -> unit_with_power','unit_expression',1,'p_unit_expression','astropy/units/format/generic.py',217),
-  ('unit_expression -> OPEN_PAREN product_of_units CLOSE_PAREN','unit_expression',3,'p_unit_expression','astropy/units/format/generic.py',218),
-  ('unit_with_power -> UNIT power numeric_power','unit_with_power',3,'p_unit_with_power','astropy/units/format/generic.py',227),
-  ('unit_with_power -> UNIT numeric_power','unit_with_power',2,'p_unit_with_power','astropy/units/format/generic.py',228),
-  ('unit_with_power -> UNIT','unit_with_power',1,'p_unit_with_power','astropy/units/format/generic.py',229),
-  ('numeric_power -> sign UINT','numeric_power',2,'p_numeric_power','astropy/units/format/generic.py',240),
-  ('numeric_power -> OPEN_PAREN paren_expr CLOSE_PAREN','numeric_power',3,'p_numeric_power','astropy/units/format/generic.py',241),
-  ('paren_expr -> sign UINT','paren_expr',2,'p_paren_expr','astropy/units/format/generic.py',250),
-  ('paren_expr -> signed_float','paren_expr',1,'p_paren_expr','astropy/units/format/generic.py',251),
-  ('paren_expr -> frac','paren_expr',1,'p_paren_expr','astropy/units/format/generic.py',252),
-  ('frac -> sign UINT division sign UINT','frac',5,'p_frac','astropy/units/format/generic.py',261),
-  ('sign -> SIGN','sign',1,'p_sign','astropy/units/format/generic.py',267),
-  ('sign -> <empty>','sign',0,'p_sign','astropy/units/format/generic.py',268),
-  ('product -> STAR','product',1,'p_product','astropy/units/format/generic.py',277),
-  ('product -> PERIOD','product',1,'p_product','astropy/units/format/generic.py',278),
-  ('division -> SOLIDUS','division',1,'p_division','astropy/units/format/generic.py',284),
-  ('power -> DOUBLE_STAR','power',1,'p_power','astropy/units/format/generic.py',290),
-  ('power -> CARET','power',1,'p_power','astropy/units/format/generic.py',291),
-  ('signed_int -> SIGN UINT','signed_int',2,'p_signed_int','astropy/units/format/generic.py',297),
-  ('signed_float -> sign UINT','signed_float',2,'p_signed_float','astropy/units/format/generic.py',303),
-  ('signed_float -> sign UFLOAT','signed_float',2,'p_signed_float','astropy/units/format/generic.py',304),
-  ('function_name -> FUNCNAME','function_name',1,'p_function_name','astropy/units/format/generic.py',310),
-  ('function -> function_name OPEN_PAREN unit_expression CLOSE_PAREN','function',4,'p_function','astropy/units/format/generic.py',316),
+  ('main -> product_of_units','main',1,'p_main','/home/mhvk/packages/astropy/astropy/units/format/generic.py',158),
```

Comment:
The prefix `/home/mhvk/packages/astropy/` needs to be removed from all of these entries, so tracebacks don't include your home directory.